src/targets/Jumper_2400.ini

```diff
@@ -24,3 +35,16 @@ extends = env:Jumper_AION_NANO_2400_TX_via_UART
 # ********************************
 # Receiver targets
 # ********************************
+
+[env:Jumper_AION_Nano_2400_RX_via_UART]
+extends = env:Unified_ESP8285_2400_RX_via_UART
+board_config = jumper.rx_2400.nano
+
+[env:Jumper_AION_Nano_2400_RX_via_BetaflightPassthrough]
+extends = env:Jumper_AION_Nano_2400_RX_via_UART
+upload_protocol = custom
+upload_speed = 420000
+upload_command = ${env_common_esp82xx.bf_upload_command}
+
+[env:Jumper_AION_Nano_2400_RX_via_WIFI]
+extends = env:Jumper_AION_Nano_2400_RX_via_UART
```

Comment:
These are no longer required we you can select the Unified TX target and then choose the layout to attach to the firmware at the end of the build process.